bot/cogs/error_handler.py

```diff
@@ -30,7 +30,8 @@ def __init__(self, bot: Bot):
         self.bot = bot
 
     @Cog.listener()
-    async def on_command_error(self, ctx: Context, e: CommandError):
+    async def on_command_error(self, ctx: Context, e: CommandError) -> None:
+        """Provide command error handling."""
```

Comment:
I think this could benefit from a more detailed docstring but perhaps that is out of scope of this PR.